README.md

```diff
@@ -1,3 +1,96 @@
-Experimental rewrite in rust of the chimera admission controller.
+> **Note well:** don't forget to checkout [Chimera's documentation](https://chimera-kube.github.io/chimera-book/)
+> for more information
 
-Leverages waPC instead of WASI to write policies.
+# policy-server
+
+`policy-server` is a
+[Kubernetes dynamic admission controller](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/)
+that uses Chimera Policies to validate admission requests.
+
+Chimera Policies are simple [WebAssembly](https://webassembly.org/)
+modules.
+
+# Deployment
+
+We recommend to rely on the [chimera-controller](https://github.com/chimera-kube/chimera-controller)
+and the [Kubernetes Custom Resources](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/)
+provided by it to deploy the Chimera stack.
+
+## Configuring policies
+
+A single instance of `policy-server` can load multiple Chimera policies. The list
+of policies to load, how to expose them and their runtime settings are handled
+through a `policies.yml` file.
```

Comment:
The `--policies` flag is required and there's no default value, so would be nice to include an example of running the command with the flag.

Reply:
fixed